dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1526,6 +1526,15 @@ a.download-video {
   }
 }
 
+#hoc_download {
+  h2 {
```

Comment:
at this point we're overriding a lot of h2, and it's not really a heading so much as an emphasised sentence with a clickable link inside it, so i would create a new class instead of overriding h2.